charts/prometheus-systemd-exporter/Chart.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: v2
+name: prometheus-systemd-exporter
+description: A Helm chart for prometheus systemd-exporter
+type: application
+version: 1.0.0
```

```suggestion
version: 0.1.0
```
semver suggests to start with 0.1.0